website/content/docs/integrations/vault-integration.mdx

```diff
@@ -22,134 +22,356 @@ install Vault separately from Nomad. Nomad does not run Vault for you.
 
 ## Agent Configuration
 
-To enable Vault integration, please see the [Nomad agent Vault
-integration][config] configuration.
+To enable Vault integration, Nomad agents must be configured with
+[`vault`][config] blocks. Each block can be enabled or disabled and configures
+how Nomad connects to the Vault cluster.
 
-## Vault Definition Syntax
+## Vault ACL
 
-To configure a job to retrieve Vault tokens, please see the [`vault` job
-specification documentation][vault-spec].
+The Vault ACL system protects the cluster from unauthorized access. When
+enabled, both Vault and Nomad must be properly configured in order for their
+integrations to work.
 
-## Using Nomad Workload Identity with Vault
+### Nomad Workload Identities
 
 Starting in Nomad 1.7, Nomad clients will use a task's [Workload Identity][] to
 authenticate to Vault and obtain a Vault token specific to the task.
 
-### Configuring Vault Authentication
+By default, Nomad only generates a workload identity for tasks that is used to
+access Nomad itself. To access Vault, jobs must define additional workload
+identities suitable for Vault access.
 
-To configure Vault for use with Nomad workload identity, you'll need to enable the Vault
-JWT backend, create a Vault role and auth method, and finally create Vault
-policies to that map Nomad workload claims to secrets in Vault.
+These additional identities can be defined as an [`identity`][] block inside
+tasks. The [`identity.name`][] value must follow the pattern
+`vault_<cluster name>`, where `<cluster name>` is the value for
+[`vault.cluster`][] of the `vault` block defined in the task or in the task
+parent group.
 
-<EnterpriseAlert>
+Support for multiple Vault clusters is only available in Nomad Enterprise. For
+Nomad Community Edition the identity name should always be `vault_default`.
+Refer to the [Multiple Vault Clusters](#multiple-vault-clusters) section for
+more information.
 
-Nomad Enterprise users who are using Vault Namespaces should
-note that you will need to pass the `-namespace` flag for each of the commands
-below. Each Vault namespace needs its own auth method, role, and
-policies.
+<Tabs>
+<Tab heading="Nomad Community Edition" group="ce">
+<CodeBlockConfig highlight="19-25">
 
-</EnterpriseAlert>
+```hcl
+job "mongo" {
+  namespace = "default"
 
-First enable the JWT authentication backend.
+  group "db" {
+    network {
+      port "db" {
+        to = 27017
+      }
+    }
 
-```shell-session
-$ vault auth enable jwt
+    task "mongo" {
+      driver = "docker"
+
+      config {
+        image = "mongo:7"
+        ports = ["db"]
+      }
+
+      vault {}
```

Comment:
The text in line 47 mentions a `vault.cluster`, but the `vault` block here is empty.

Reply:
Yeah, `cluster` is optional and only available in Nomad Enterprise.